lib/fs.js

```diff
@@ -43,6 +43,18 @@ const internalUtil = require('internal/util');
 const assertEncoding = internalFS.assertEncoding;
 const stringToFlags = internalFS.stringToFlags;
 const getPathFromURL = internalURL.getPathFromURL;
+const O_RDONLY = constants.O_RDONLY | 0;
```

Comment:
@mscdex it gives a hint to the compiler that it's a SMI. 

I'm just wondering why it isn't done on the constant itself. I guess that since this is copied over it shouldn't be a big deal here.